arangod/V8Server/v8-collection.cpp

```diff
@@ -2570,8 +2570,6 @@ static void JS_CompletionsVocbase(
       .FromMaybe(false);
   result->Set(context, j++, TRI_V8_ASCII_STRING(isolate, "_dropView()"))
       .FromMaybe(false);
-  result->Set(context, j++, TRI_V8_ASCII_STRING(isolate, "_engine()"))
```

Comment:
If this isn't removed here, the console auto-completion shows `db._engine()` twice.
I don't know why exactly, but removing this line here fixes it.

Reply:
don't know if has something to do with this happening, but there is one  
`TRI_AddMethodVocbase(isolate, ArangoNS,
                       TRI_V8_ASCII_STRING(isolate, "_engine"), JS_Engine);` 
in `arangod/V8Server/v8-vocbase.cpp` 
maybe not because this is not setting anything to the result

Reply:
Yes, I saw this. But for other method names occurring in JS_CompletionsVocbase there is also an equivalent call to `TRI_AddMethodVocbase(...)`, for example for `_engineStats`. But `_engineStats` does not occur twice during auto-completion. That's why I mentioned I don't know why.